integration/spark/spark32/build.gradle

```diff
@@ -53,6 +53,7 @@ dependencies {
     testAnnotationProcessor "org.projectlombok:lombok:${lombokVersion}"
 
     implementation(project(":shared"))
+    implementation(project(":spark3"))
```

Comment:
I think it's better to specify dependencies manually here - it simplifies our internal dependency graph

Reply:
can we do it different way? here we need stuff from `io.openlineage.spark3.agent.lifecycle.plan.catalog` and `io.openlineage.spark3.agent.utils`